lib/elixir/lib/uri.ex

```diff
@@ -941,6 +941,28 @@ defmodule URI do
   defp path_to_segments(path) do
     path |> String.split("/") |> Enum.reverse()
   end
+
+  @doc """
+  Appends `query` to the given `uri`.
+
+  The given `query` is not automatically encoded, use `encode/2` or `encode_www_form/1`.
+
+  ## Examples
+
+      iex> URI.append_query(URI.parse("http://example.com/"), "x=1") |> URI.to_string()
+      "http://example.com/?x=1"
+
+      iex> URI.append_query(URI.parse("http://example.com/?x=1"), "x=2") |> URI.to_string()
+      "http://example.com/?x=1&x=2"
```

Comment:
Nit: Wouldn't it be better to use a second different query param name in the example? Such as `x` and `y`, or maybe `foo` and `bar`, so the example is more realistic (when the `x` query param is not set twice)

Reply:
Yeah that was my initial idea however I decided to do this to showcase that there is no automatic deduplication which someone could maybe expect. No strong opinion either way!

Reply:
@wojtekmach my 2c: let's have both examples. No deduplication is interesting to showcase but a non-duplicated example would be nice too.